### ISSUE TITLE
MangaDex upload

### ISSUE THREAD

**Issue by Albirew** [TorchScans/Ghost-Gyaru-#1]
Hi, since dex is back, is it possible to upload chapters 7 to 30 over there?
https://mangadex.org/title/6316eac8-53cc-4ac8-bc6b-7c5e229ac607/upload

**Comment by TorchScans** [comment]
Yeah, I will do that, but once the UI is a bit more user-friendly, and
allows editing/deleting chapters.

**Comment by Albirew** [comment]
Oh, i see... i just checked and even as group leader, i still can't edit or delete uploaded chapters...
Well, i will use this issue to notify you when this become possible (or you'll notify me by closing this issue, same result)

**Comment by TorchScans** [comment]
Alright.
